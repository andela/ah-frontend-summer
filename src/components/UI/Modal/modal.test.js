import React from 'react';
import { shallow } from 'enzyme';

import Modal from './Modal';

describe('tests Modal component', () => {
    const props = {
        modalClosed: jest.fn(),
        show: false,
        children: null
    };
    let wrapper = shallow(<Modal {...props} />);

    it('should render with out crashing', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
