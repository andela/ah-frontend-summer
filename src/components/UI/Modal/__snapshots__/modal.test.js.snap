// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests Modal component should render with out crashing 1`] = `
<Fragment>
  <backdrop
    clicked={[MockFunction]}
    show={false}
  />
  <div
    className="Modal"
    style={
      Object {
        "opacity": "0",
        "transform": "translateY(-100vh)",
      }
    }
  />
</Fragment>
`;
